opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -234,8 +246,7 @@ mod tests {
             .with_simple_exporter(exporter.clone())
             .build();
 
-        let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
-        let subscriber = tracing_subscriber::registry().with(layer);
+        let subscriber = create_tracing_subscriber(exporter.clone(), &logger_provider);
```

Comment:
why is exporter required to be passed?

Reply:
Forgot to remove as part of updating the method. Will cleanup in subsequent PR.